src/app/globals.css

```diff
@@ -4,56 +4,56 @@
 
 @font-face {
```

Comment:
This is not the correct way of adding google fonts. You just need to add the URL at the top of the CSS file and then set the font family at the root level of the project.